README.md

```diff
@@ -323,6 +323,7 @@ concurrently can be used programmatically by using the API documented below:
   - `prefixColors`: a list of colors as supported by [chalk](https://www.npmjs.com/package/chalk).
     If concurrently would run more commands than there are colors, the last color is repeated.
     Prefix colors specified per-command take precedence over this list.
+  - `colors`: let colours be selected to vary automatically where not explicitly defined
```

Comment:
For consistency with the option name itself

```suggestion
  - `colors`: let colors be selected to vary automatically where not explicitly defined
```

Reply:
Also I think we should name the option more precise, so that it is clear that only the prefixes are affected by it.

Reply:
following the feedback I've opted to include this functionality as part of the `prefixColors` option, however the auto color functionality is opt-in so: 
- if someone wants all auto colors they just do `concurrently -c auto` 
- if they want to specify some colors but want the rest to be auto they can do so with the same option e.g. `concurrently -c red,green,auto,red,auto`
- if they want the current behaviour of repeating the last color then they just need to omit the last `auto` e.g. `concurrently -c red,green,auto,red`

Reply:
docs text tbc, what do you guys think?